locales/en.default.schema.json

```diff
@@ -827,6 +827,10 @@
             "show_dynamic_checkout": {
               "label": "Show dynamic checkout buttons",
               "info": "Using the payment methods available on your store, customers see their preferred option, like PayPal or Apple Pay. [Learn more](https://help.shopify.com/manual/using-themes/change-the-layout/dynamic-checkout)"
+            },
+            "show_gift_card_recipient": {
+              "label": "Show recipient form for gift card products",
+              "info": "When enabled, gift card products can optionally be sent to a recipient with a personal message."
```

Comment:
I think ideally here we would link to our help doc on how to add a gift card product: https://help.shopify.com/en/manual/products/gift-card-products/add-update-gift-card-products

Though doing it now means re requesting translations.